### ISSUE TITLE
Insights about improving accuracy for MNIST dataset

### ISSUE THREAD

**Issue by Shadab442** [stefanonardo/pytorch-esn#22]
If I want to improve the accuracy (say around 95%) for classification on the MNIST dataset, do I need to tune the hyperparameters? Or do I need to change anything in the implementation? Can you give some insights about what I may need to improve?
